satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -82,255 +82,316 @@ datasets:
     name: quality_rtm_ctth
     resolution: 2000
     file_type: nc_fci_ctth
     file_key: quality_rtm
+    standard_name: status_flag
+    fill_value: -127
+    import_enum_information: True
 
   quality_method_ctth:
     name: quality_method_ctth
     resolution: 2000
     file_type: nc_fci_ctth
     file_key: quality_method
+    standard_name: status_flag
+    fill_value: -127
+    import_enum_information: True
 
   quality_nwp_parameters_ctth:
     name: quality_nwp_parameters_ctth
     resolution: 2000
     file_type: nc_fci_ctth
     file_key: quality_nwp_parameters
+    standard_name: status_flag
+    fill_value: -127
+    import_enum_information: True
 
   quality_MTG_parameters_ctth:
     name: quality_MTG_parameters_ctth
     resolution: 2000
     file_type: nc_fci_ctth
     file_key: quality_MTG_parameters
+    standard_name: status_flag
     fill_value: -127
+    import_enum_information: True
 
   quality_overall_processing_ctth:
     name: quality_overall_processing_ctth
     resolution: 2000
     file_type: nc_fci_ctth
     file_key: quality_overall_processing
+    standard_name: quality_flag
+    fill_value: -127
+    import_enum_information: True
 
   quality_overall_processing_aviation_ctth:
     name: quality_overall_processing_aviation_ctth
     resolution: 2000
     file_type: nc_fci_ctth
     file_key: quality_overall_processing_aviation
+    standard_name: quality_flag
+    fill_value: -127
+    import_enum_information: True
 
   product_quality_ctth:
     name: product_quality_ctth
     file_type: nc_fci_ctth
     file_key: product_quality
-    long_name: product_quality_index
+    standard_name: product_quality
 
   product_completeness_ctth:
     name: product_completeness_ctth
     file_type: nc_fci_ctth
     file_key: product_completeness
-    long_name: product_completeness_index
+    standard_name: product_completeness
 
   product_timeliness_ctth:
     name: product_timeliness_ctth
     file_type: nc_fci_ctth
     file_key: product_timeliness
-    long_name: product_timeliness_index
+    standard_name: product_timeliness
 
   # OCA
   retrieved_cloud_phase:
     name: retrieved_cloud_phase
     resolution: 2000
     file_type: nc_fci_oca
     file_key: retrieved_cloud_phase
-    standard_name: thermodynamic_phase_of_cloud_water_particles_at_cloud_top
+    standard_name: thermodynamic_phase_of_cloud_particles_classification
+    fill_value: -127
+    import_enum_information: True
 
   retrieved_cloud_optical_thickness:
     name: retrieved_cloud_optical_thickness
     resolution: 2000
     file_type: nc_fci_oca
     file_key: retrieved_cloud_optical_thickness
-    long_name: cloud_optical_depth
+    standard_name: atmosphere_optical_thickness_due_to_cloud
 
   retrieved_cloud_optical_thickness_upper_layer:
     name: retrieved_cloud_optical_thickness_upper_layer
     resolution: 2000
     file_type: nc_fci_oca
     file_key: retrieved_cloud_optical_thickness
     layer: 0
-    long_name: cloud_optical_depth
```

Comment:
Here I would add the long name from the FCIL2FS but with the addition `"(upper layer)"` at the end

Reply:
appended  "for upper/lower layer" to the long names